src/rust/engine/fs/store/src/snapshot.rs

```diff
@@ -538,9 +538,11 @@ impl Snapshot {
     future::result(digest_hint.ok_or_else(|| "No digest hint provided.".to_string()))
       .and_then(move |digest| Snapshot::from_digest(store, digest, workunit_store))
       .or_else(|_| {
+        let use_gitignore = false;
```

Comment:
Iirc, this is only used by V1 code? V2 code that uses `Snapshot`s will use Gitignore?